libs/application/templates/financial-statement-cemetery/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,236 @@
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  total: z.string().refine((x) => !!x, { params: m.required }),
+})
+
+const cemeteryAsset = z.object({
+  currentAssets: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  fixedAssetsTotal: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  total: z.string(),
+})
+
+const cemeteryIncome = z.object({
+  careIncome: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  burialRevenue: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  grantFromTheCemeteryFund: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  otherIncome: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  total: z.string(),
+})
+
+const cemeteryExpense = z.object({
+  payroll: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  funeralCost: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  chapelExpense: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  donationsToOther: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  cemeteryFundExpense: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  otherOperationCost: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  depreciation: z
+    .string()
+    .refine((x) => !!x, { params: m.required })
+    .refine((x) => checkIfNegative(x), { params: m.negativeNumberError }),
+  total: z.string(),
+})
+
+const about = z.object({
+  nationalId: z
+    .string()
+    .refine((val) => (val ? kennitala.isValid(val) : false), {
+      params: m.nationalIdError,
+    }),
+  fullName: z.string().refine((x) => !!x, { params: m.required }),
+  powerOfAttorneyNationalId: z.string().optional(),
+  powerOfAttorneyName: z.string().optional(),
+  phoneNumber: z.string().refine(
+    (p) => {
+      // ignore validation on dev to test with Gervimenn remove b4 release
+      if (isRunningOnEnvironment('dev')) {
+        return true
+      }
```

Comment:
Remove?

Reply:
Probably, this is in the inao version as well...